packages/components/src/components/input/input.scss

```diff
@@ -38,7 +38,7 @@
             outline: none;
 
             & ~ label {
-                transform: translate(0, -1.6rem) scale(0.75);
+                transform: translate(0, -1.8rem) scale(0.75);
```

Comment:
@zaki-binary can you pls add to description what did you fix here. cause its being used other places as well

Reply:
It's just to fix the vertical alignment issue for the placeholder label to title transformation, currently the alignment for `label` when it is transformed on focus or active is slightly mis-aligned. I added a screen of this issue and mentioned I was going to fix this on the private channel.